stdlib/asm/crypto/stark/verifier.masm

```diff
@@ -141,7 +140,7 @@ export.verify
     #       VI) Evaluate the constraints over the OOD frame and assert equality with H(z)
     #==============================================================================================
 
-    # TODO: Compare with the evaluation of the constraints on the EvaluationFrame
+    # Compare with the evaluation of the constraints on the trace columns OOD evaluation frame. 
```

Comment:
Shouldn't the `TDDO` stay here?

Reply:
We usually remove todos when we have open issues tracking them. Re-added it now.